addons/io_scene_gltf/camera.py

```diff
@@ -2,7 +2,23 @@
 
 
 def create_camera(op, idx):
-    camera = op.gltf['cameras'][idx]
-    name = camera.get('name', 'cameras[%d]' % idx)
-    data = bpy.data.cameras.new(name)
-    return data
+    """
+    create a new blender camera from the gltf data
+    """
+    data = op.gltf['cameras'][idx]
+    name = data.get('name', 'cameras[%d]' % idx)
+    camera = bpy.data.cameras.new(name)
+    if data["type"] == "orthographic":
+        camera.type = "ORTHO"
+        p = data["orthographic"]
+        camera.clip_start = p["znear"]
+        camera.clip_end = p["zfar"]
+        camera.ortho_scale = max(p["xmag"], p["ymag"])
+    else:
+        camera.type = "PERSP"
+        p = data["perspective"]
+        camera.clip_start = p["znear"]
+        camera.clip_end = p["zfar"]
+        camera.lens_unit = "FOV"
+        camera.angle_x = p["yfov"] * p["aspectRatio"]
```

Comment:
And `aspectRatio` isn't required. Apparently when it's not there, "the aspect ratio of the canvas is used". Not sure what to do about that. Maybe just pick a default value?

Reply:
Great!

Currently, tests fail due to test scenes (VC.gltf for instance) that do not have the aspectRation set.

Maybe something like:
canvas_aspect_ratio = bpy.context.scene.render.resolution_y / bpy.context.scene.render.resolution_x
p.get("aspectRation", canvas_aspect_ratio)

Reply:
In the updated PR I update the angle_x instead of angle_y according to the presence of apectRatio